scene/resources/texture.h

```diff
@@ -1093,4 +1093,29 @@ class PlaceholderCubemapArray : public PlaceholderTextureLayered {
 			PlaceholderTextureLayered(LAYERED_TYPE_CUBEMAP_ARRAY) {}
 };
 
+class Texture2DRD : public Texture2D {
```

Comment:
I think these classes should probably be put in the servers/rendering/renderer_rd, so if this back-end is disabled, these textures are disabled too.

Reply:
That makes sense